samples/Vim Help File/modeline.txt

```diff
@@ -0,0 +1 @@
+ vim:tw=78:ts=8:noet:ft=help:norl:
```

Comment:
Could you replace this with a more realistic sample files?

Reply:
Uh, no. 😜 That'll cause random `.txt` files to be misclassified as "Vim Help"...

Reply:
How is that?

Reply:
Because depending on what sample I use, Linguist is going to disambiguate two `.txt` files based on whichever words they most have in common with the sample. Adding a file with a modeline in it is the least disruptive thing I could think of.

Reply:
> Linguist is going to disambiguate two `.txt` files based on whichever words they most have in common with the sample.

Unless I'm missing something, this is only true if we're using the Bayesian classifier. Since you have a fallback to Text in your heuristic rule, we will never use the Bayesian classifier.

Reply:
Then what's the point of having a realistic sample?

Reply:
Sorry for the long delay.

Ok, I tried to think of other solutions to avoid having unrealistic files in `samples/`, but can't think of a better way :/
Could you however add a comment in the heuristic rule above the fallback to explain this (i.e., state that the `Text` fallback shouldn't be removed because we have unrealistic samples and can't rely on the Bayesian classifier)? I just want to make sure we don't end up using unrealistic samples in the future.

Reply:
I'll, uh, just leave a permalink to that comment…